src/app/scroll-collapse/scroll-collapse.directive.ts

```diff
@@ -220,6 +233,7 @@ export class ScrollCollapseDirective implements AfterViewInit, OnDestroy {
    */
   public calculateAffixMode(viewport: Viewport): void {
     this.affixMode = viewport.scrollY >= this.originalTop - this.yOffset;
+    this.affixChange.emit(this.affixMode);
```

Comment:
should this only emit if there is a change?

Reply:
Updated